docs/Design/Design-Guide.md

```diff
@@ -6,20 +6,18 @@
 - [Accessibility](#accessibility)
```

Comment:
This file actually doesn't change anything. The PR went into raft a little differently than how it went into HHS, so Git sees it as a change, while the content is the same